File: 01-Ejercicios-Clases-2023_2/C8-09-01-23--09-08-23/ProyectosJAVA/LP2SoftRRHHModel/src/pe/edu/pucp/lp2soft/rrhh/model/Persona.java
package pe.edu.pucp.lp2soft.rrhh.model;
import java.util.Date;
public class Persona {
    private int idPersona;
    private String DNI;
    private String nombre;
    private String apellidoPaterno;
    private char genero;
    private Date fechaNacimiento;

    public Persona(String DNI, String nombre, String apellidoPaterno, char genero, Date fechaNacimiento) {
        this.DNI = DNI;
        this.nombre = nombre;
        this.apellidoPaterno = apellidoPaterno;
        this.genero = genero;
        this.fechaNacimiento = fechaNacimiento;
    }

    
    
    public int getIdPersona() {
        return idPersona;
    }

    public void setIdPersona(int idPersona) {
        this.idPersona = idPersona;
    }

    public String getDNI() {
        return DNI;
    }

    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidoPaterno() {
        return apellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        this.apellidoPaterno = apellidoPaterno;
    }

    public char getGenero() {
        return genero;
    }

    public void setGenero(char genero) {
        this.genero = genero;
    }

    public Date getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(Date fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }
    
    
}

File: 04-Codigo-Clases/BackCSharp_FrontJava/CSharp/LP2Soft/LP2SoftRRHHModel/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftRRHHModel
{
    public class Persona
    {
        private int _idPersona;
        private string _DNI;
        private string _nombre;
        private string _apellidoPaterno;
        private char _genero;
        private DateTime _fechaNacimiento;

        public int IdPersona { get => _idPersona; set => _idPersona = value; }
        public string DNI { get => _DNI; set => _DNI = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string ApellidoPaterno { get => _apellidoPaterno; set => _apellidoPaterno = value; }
        public char Genero { get => _genero; set => _genero = value; }
        public DateTime FechaNacimiento { get => _fechaNacimiento; set => _fechaNacimiento = value; }
    }
}
